src/app/zap-templates/zcl/data-model/chip/general-diagnostics-cluster.xml

```diff
@@ -92,6 +92,7 @@ limitations under the License.
     <attribute side="server" code="0x06" define="ACTIVE_RADIO_FAULTS" type="ARRAY" entryType="RadioFaultEnum" writable="false" optional="true">ActiveRadioFaults</attribute>
     <attribute side="server" code="0x07" define="ACTIVE_NETWORK_FAULTS" type="ARRAY" entryType="NetworkFaultEnum" writable="false" optional="true">ActiveNetworkFaults</attribute>
     <attribute side="server" code="0x08" define="TEST_EVENT_TRIGGERS_ENABLED" type="BOOLEAN" writable="false" optional="false">TestEventTriggersEnabled</attribute>
+    <attribute side="server" code="0x09" define="AVERAGE_WEAR_COUNT" type="INT32U" min="0x00000000" max="0xFFFFFFFF" writable="false" default="0x00000000" optional="true">AverageWearCount</attribute>
```

Comment:
since this is a INT32U do we need min/max that goes over the entire range?

Reply:
I've used `TotalOperationalHours` as reference. I'm not sure how to choose a max value to restrict the range.

Reply:
This is not restricting the range, right?  So the min/max are just noise....

Reply:
Removed the redundant, "noisy" ranges from attributes of type integer.